birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -67,4 +67,7 @@ c.JupyterHub.log_level = logging.DEBUG
 
 c.Spawner.debug = True
 
+## Extra arguments to be passed to the single-user server.
+c.Spawner.args = ["--NotebookApp.terminals_enabled=False"]
```

Comment:
The terminal is disabled, but the user can still launch bash commands from a notebook... If the terminal is not used, you're right that it should be disabled.

Reply:
You're right, this just make it less obvious to get to the terminal and we have no use-case for the terminal yet.